TokenSdk.xcodeproj/project.pbxproj

```diff
@@ -1092,9 +1092,12 @@
 			files = (
 			);
 			inputPaths = (
+				"${PODS_PODFILE_DIR_PATH}/Podfile.lock",
```

Comment:
I didn't try to understand any of the changes in this file. If you were hoping that that someone would double-check this part of the change, I can try harder :-)

Reply:
it is autogenerated by the xcode, we dont need to worry about this file